Redefine (co)limits in terms of kan extensions (#186)
plt-amy/1lab: src/Cat/Instances/Shape/Initial.lagda.md

```agda
open import 1Lab.Prelude

open import Cat.Base

module Cat.Instances.Shape.Initial where
```

<!--
```agda
open Precategory
```
-->

The **initial category** is the category with no objects.

```agda
⊥Cat : Precategory lzero lzero
⊥Cat .Ob = ⊥
⊥Cat .Hom _ _ = ⊥
```

> This category is notable, as there exists a unique functor from

Reviewer: ```suggestion
This category is notable for the existence of a unique functor from
```
